frontend/src/features/frame/components/topbar/AccountInfo.tsx

```diff
@@ -22,6 +23,16 @@ const LoggedInActions = () => {
 
   const { logout, loginState } = useLoginState();
 
+  const [menuAnchor, setMenuAnchor] = React.useState<null | HTMLElement>(null);
```

Comment:
I'm not familiar with the fact that having an anchor means that the menu is open and visible. To me the anchor is where the menu will be, not whether it is closed or open. Would it be worth it to add one line of comment above this state to explain its purpose?

As I read the code below, I would even recommned the refactor:
- Anchor is not a state that changes, but is passed as a props (in this case the top-right button in the header)
- `PersonalMenu` takes open as a props
- The state here is replaced by `isMenuOpen`

Unless you have reasons not to?

Reply:
Thanks for the review.

The anchor is indeed not a state that changes during the life cycle of the component. I used it because I followed the MUI documentation, and I agree we can simplify the code by relaying only on `open`.

I'll refactor it before merging.

Reply:
> Unless you have reasons not to?

I made the refactor. I find the new code slightly more complex now, as it have two independent notions, anchor and open, nevertheless more readable with the presence of the open notion in the parent component.

Can a menu be open without having an anchor? The original implementation didn't allow this as the open state was deduced directly by reading the anchor prop.

I feel equally comfortable with the new and the original implementation. I like the fact the original one has less code, and I like the readability of the new one.

Unless objection @lfaucon , I'll merge it as is (with the new implementation).